addons/pos_paytm/static/src/js/payment_paytm.js

```diff
@@ -0,0 +1,146 @@
+odoo.define('pos_paytm.payment', function (require) {
+"use strict";
+
+const core = require('web.core');
+const rpc = require('web.rpc');
+const PaymentInterface = require('point_of_sale.PaymentInterface');
+const { Gui } = require('point_of_sale.Gui');
+
+const _t = core._t;
+
+let PaymentPaytm = PaymentInterface.extend({
+    init: function (pos, payment_method) {
+        this._super(...arguments);
+    },
+
+    send_payment_request: async function (cid) {
+        /**
+         * Override
+         */
+        this._super.apply(this, arguments);
+        const line = this.pos.get_order().selected_paymentline;
+        const order = this.pos.selectedOrder;
+        let retry = localStorage.getItem(order.uid);
```

Comment:
I don't understand this: `orderId`
You send it through some funtion and become `merchantTransactionId` in `_paytm_get_request_body`
it is necessary to be consistent in the way of noting its variables
and why not just use the `pos.order`?
should the number be unique?
In this case add the `cid`?